internal/generator/vector/output/kafka/kafka_test.go

```diff
@@ -59,8 +59,8 @@ timestamp_format = "rfc3339"
 
 # Sasl Config
 [sinks.kafka_receiver.sasl]
-username = "" # inject code to read from filename
-password = "" # inject code to read from filename
+user = ""
```

Comment:
please add tests which generate proper values of username/password

Reply:
same for elasticsearch

Reply:
Conf generator reads user/password from `/var/run/ocp-collector/secrets` but in case of unit test case this file doesn't exist so user/password is empty in unit test case. Even incase of fluentd value is `username "#{File.exists?('/var/run/ocp-collector/secrets/kafka-receiver-1/username') ? open('/var/run/ocp-collector/secrets/kafka-receiver-1/username','r') do |f|f.read end : ''}"`